libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -782,16 +755,20 @@ ContractLevelChecker::FunctionSet const& ContractLevelChecker::getBaseFunctions(
 	if (result != m_contractBaseFunctions.cend())
 		return result->second;
 
-	FunctionSet set;
+	FunctionSet set, tmpSet;
 
 	for (auto const* base: resolveBaseContracts(*_contract))
-//	{
+	{
 		for (auto const* func: base->definedFunctions())
```

Comment:
Why not just have ``tmpSet`` local to the loop and create it as
``FunctionSet tmpSet{base->definedFunctions().begin(), base->definedFunctions().end()};``` instead of inserting here?